eq-author/src/components/CommentsPanel/CommentHeader.js

```diff
@@ -0,0 +1,108 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+import moment from "moment";
+import "moment/locale/en-gb";
+
+import {
+  CommentHeaderContainer,
+  AvatarWrapper,
+  AvatarOuter,
+  AvatarInner,
+  NameWrapper,
+  DateField,
+  EditButton,
+  DeleteComment,
+  ToolTipWrapper,
+} from "./index";
+
+export const FlexLabel = styled.div`
+  font-size: 1em;
+  align-items: center;
+  height: 20px;
+  overflow: hidden;
+  white-space: nowrap;
+  width: 180px;
+
+  @media (max-width: 1700px) {
+    width: 138px;
+  }
+
+  @media (max-width: 1500px) {
+    width: 120px;
+    font-size: 0.9em;
+  }
+  text-overflow: ellipsis;
+`;
+
+const FlexLabelReplies = styled(FlexLabel)`
+  @media (max-width: 1700px) {
+    width: 107px;
+  }
+`;
+
+const CommentHeader = props => {
+  const {
+    myId,
+    getInitials,
+    index,
+    repliesIndex,
+    repliesItem,
+    shared,
+    item,
+    handleEdit,
+    handleDelete,
+  } = props;
+
+  return (
+    <>
+      <CommentHeaderContainer>
+        <AvatarWrapper>
+          <AvatarOuter avatarColor={shared.user.id === myId}>
+            <AvatarInner>{getInitials(shared.user.name)}</AvatarInner>
+          </AvatarOuter>
+        </AvatarWrapper>
+        <NameWrapper>
+          <FlexLabelReplies>{shared.user.displayName}</FlexLabelReplies>
+          <DateField>{moment(shared.createdTime).calendar()}</DateField>
+        </NameWrapper>
+        <ToolTipWrapper content={"Edit"}>
+          <EditButton
```

Comment:
Could this be renamed to **EditComment** so that it is more consistent with the **DeleteComment** button?